QuestionSets/PythonLanguage/BasicsAndArrays/Manan_Garg/sol1_manan.py

```diff
@@ -0,0 +1,5 @@
+Array = []
+A = range(0,1000)
+for i in A:
+    Array.append(i+1)
+print (Array)
```

Comment:
Is there a way where you can have the same output in one line?

HINT: _you already have the one-line solution in your code somewhere._